modules/overrides/overrides.go

```diff
@@ -58,12 +59,12 @@ func NewOverrides(defaults Limits) (*Overrides, error) {
 	subservices := []services.Service(nil)
 
 	if defaults.PerTenantOverrideConfig != "" {
-		runtimeCfg := runtimeconfig.ManagerConfig{
+		runtimeCfg := runtimeconfig.Config{
 			LoadPath:     defaults.PerTenantOverrideConfig,
 			ReloadPeriod: time.Duration(defaults.PerTenantOverridePeriod),
 			Loader:       loadPerTenantOverrides,
 		}
-		runtimeCfgMgr, err := runtimeconfig.NewRuntimeConfigManager(runtimeCfg, prometheus.DefaultRegisterer)
+		runtimeCfgMgr, err := runtimeconfig.New(runtimeCfg, prometheus.WrapRegistererWithPrefix("cortex_", prometheus.DefaultRegisterer), log.Logger)
```

Comment:
NOTE: Keeping the prefix `cortex_` as I find this metric `cortex_runtime_config_last_reload_successful` used in one of the tempo alerts.

```
ksonnet/lib/tempo-cloud-mixin/alerts.libsonnet
36:                cortex_runtime_config_last_reload_successful{namespace=~"%s"} == 0
```

Reply:
Can we update that alert instead?

Reply:
Hmm. But we couldn't change the alert until this change reaches the particular cell correct?

What we can do is, I can change prefix to `tempo_` and once this reaches the prod, someone can change the alert name. WDYT?

Reply:
Ah. the alert is from vendor. I can directly change it in here!. 👍

Reply:
I changed the prefix to `tempo_`. and created TODO https://github.com/grafana/deployment_tools/pull/15791
for the reminder!

Reply:
Hmm, shouldn't that alert be in the public mixin because its not specific to Grafana Cloud?

Reply:
But this is fine, can be picked in another PR :)